lib/renaissance/bids/bids.ex

```diff
@@ -0,0 +1,53 @@
+defmodule Renaissance.Bids do
+  import Ecto.Query
+  alias Renaissance.{Repo, Bid}
+  alias Renaissance.Helpers.Adapt
+
+  def place_bid(params) do
+    params = Adapt.format_amount(params, "amount")
+
+    Bid.changeset(%Bid{}, params)
+    |> Repo.insert()
+  end
+
+  def exists?(id) do
+    Repo.exists?(from b in Bid, where: b.id == ^id)
+  end
+
+  def get_highest_bid(auction_id) do
+    query = "SELECT bidder_id, amount, created_at
```

Comment:
This query should be constructed through Ecto: https://hexdocs.pm/ecto/Ecto.Query.html

Reply:
Resolved in 845f45e.